feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/component/FloorMapSideEventItem.kt

```diff
@@ -1,18 +1,146 @@
+import io.github.droidkaigi.confsched2023.model.SideEvent.Mark.Mark1
+import io.github.droidkaigi.confsched2023.model.SideEvents
 
 @Composable
 fun FloorMapSideEventItem(
     sideEvent: SideEvent,
     onSideEventClick: (SideEvent) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    Column(modifier) {
-        Text(sideEvent.toString())
+    Column(
+        modifier
+            .clickable { onSideEventClick(sideEvent) }
+            .padding(horizontal = 16.dp)
+    ) {
+        Spacer(modifier = Modifier.height(16.dp))
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            val (iconRes, iconColor) = sideEvent.mark.iconResAndColor()
+            Icon(
+                painter = painterResource(id = iconRes),
+                contentDescription = FavoriteIcon.asString(),
+                tint = iconColor,
+                modifier = Modifier.size(16.dp),
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = sideEvent.title.currentLangTitle,
+                style = MaterialTheme.typography.labelLarge,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+            )
+        }
+        Spacer(modifier = Modifier.height(4.dp))
+        Text(
+            text = sideEvent.description.currentLangTitle,
+            style = MaterialTheme.typography.bodyMedium,
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_schedule),
+                contentDescription = FavoriteIcon.asString(),
+                modifier = Modifier.size(16.dp)
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = sideEvent.timeText.currentLangTitle,
+                style = MaterialTheme.typography.bodySmall,
+            )
+        }
+        if (sideEvent.link != null) {
+            Spacer(modifier = Modifier.height(8.dp))
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_link),
+                    contentDescription = FavoriteIcon.asString(),
+                    modifier = Modifier.size(16.dp)
+                )
+                Spacer(modifier = Modifier.width(8.dp))
+                Text(
+                    text = createAnnotatedEventDetailString(),
+                    style = MaterialTheme.typography.bodySmall,
+                    modifier = Modifier.clickable {
+                        // todo open link
+                    }
+                )
+            }
+        }
+        Spacer(modifier = Modifier.height(16.dp))
+        Divider()
+    }
+}
+
+@Composable
+private fun Mark.iconResAndColor(): Pair<Int, Color> {
+    return when (this) {
+        Mark1 -> R.drawable.ic_favorite to LocalContentColor.current
```

Comment:
I tried to define the color of the icon, but couldn't find the right place.
For example, I thought it would be good to define pink in designsystem/theme/Color.kt? (It didn't seem to be visible in the visibility of the module)
In the sense of temporary, if you directly define #DC369A here, it will be resolved.